ingestion/src/metadata/ingestion/sink/elasticsearch.py

```diff
@@ -538,6 +554,61 @@ def _create_team_es_doc(self, team: Team):
 
         return team_doc
 
+    def _create_glossary_es_doc(self, glossary: Glossary):
+        fqdn = glossary.fullyQualifiedName
+        suggest = [
+            {
+                "input": [
+                    glossary.displayName if glossary.displayName else glossary.name
+                ],
+                "weight": 10,
+            }
+        ]
+        tags = set()
+        timestamp = time.time()
+        glossary_owner = (
+            str(glossary.owner.id.__root__) if glossary.owner is not None else ""
+        )
+        glossary_followers = []
+        if glossary.followers:
+            for follower in glossary.followers.__root__:
+                glossary_followers.append(str(follower.id.__root__))
+        tier = None
+        for glossary_tag in glossary.tags:
+            if "Tier" in glossary_tag.tagFQN:
+                tier = glossary_tag.tagFQN
+            else:
+                tags.add(glossary_tag.tagFQN)
+        # tasks: List[Task] = glossary.tasks  # TODO Handle Glossary words
```

Comment:
Remove commented code?

Reply:
This code looks bunch of work. Will address it in subsequent patches.